pkgs/by-name/hq/hqplayerd/package.nix

```diff
@@ -7,36 +7,34 @@
   cairo,
   fetchurl,
   flac,
-  gcc12,
+  gcc,
```

Comment:
IMO it's better to always used a pinned gcc variant so bumps to gcc don't break this package.